doc/nrf/ug_zigbee_configuring_components.rst

```diff
@@ -0,0 +1,31 @@
+﻿.. _ug_zigbee_configuring_components:
+
+Configuring Zigbee libraries in |NCS|
+#####################################
+
+The Zigbee protocol in |NCS| can be customized by enabling and configuring several libraries.
+
+This page lists options and steps required for configuring the available libraries:
+
+.. contents::
+    :local:
+    :depth: 1
+
+.. _ug_zigbee_configuring_components_ota:
+
+Configuring default signal handler
+**********************************
+
+The default signal handler provides a default logic to handle ZBOSS stack signals.
+For more information, see :ref:`lib_zigbee_signal_handler`.
+
+The default signal handler can be enabled and configured using the following Kconfig options:
```

Comment:
No Kconfig options available. It is used by calling the `zigbee_default_signal_handler` inside aplication's `zboss_zignal_handler` implementation.

Reply:
Reworked this part.